packages/apputils-extension/src/notificationplugin.tsx

```diff
@@ -217,16 +223,11 @@ function NotificationCenter(props: INotificationCenterProps): JSX.Element {
                         )}
                       </div>
                     </div>
-                    <button
-                      className={`jp-Button jp-mod-minimal ${TOAST_CLOSE_BUTTON_CLASS}`}
-                      title={trans.__('Dismiss notification')}
-                      onClick={closeNotification}
-                    >
-                      <deleteIcon.react
-                        className="jp-icon-hover"
-                        tag="span"
-                      ></deleteIcon.react>
-                    </button>
+                    <Private.CloseButton
+                      close={closeNotification}
+                      closeIcon={deleteIcon.react}
+                      closeIconMargin
+                    />
```

Comment:
Tests are failing which appears to be related to this change:

```js
 1) [jupyterlab] › test/jupyterlab/notification.test.ts:258:7 › Notification center › should forget dismissed notification 

    Retry #1 ───────────────────────────────────────────────────────────────────────────────────────
    Error: locator.click: Target closed
    =========================== logs ===========================
    waiting for locator('.jp-Notification-List').locator('li').locator('[title="Dismiss notification"]').first()
    ============================================================

      278 |       .locator('.jp-Notification-List >> li >> [title="Dismiss notification"]')
      279 |       .first()
    > 280 |       .click();
          |        ^
      281 |
      282 |     await expect(status).toHaveText('1');
      283 |   });

        at /home/runner/work/jupyterlab/jupyterlab/galata/test/jupyterlab/notification.test.ts:280:8
```

Reply:
CC @andrii-i - sorry, for two reviews I missed the tests at first glance (assumed those are flaky ones)

Reply:
@krassowski No problems, thank you for catching this, will give it a look

Reply:
Fixed